### PR TITLE
Code 146 resolve cc issues - part 1

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
#### Which User story does this relate to (link)?
-  https://osi-cwds.atlassian.net/browse/CODE-148

#### Brief feature description
- Resolve code climate code smells and duplication for County Admin.

### CODE REVIEW

File: app/javascript/_utils/checkAdminRoles.js [hunk 1]
```diff
@@ -2,9 +2,7 @@ export const getAdminOfficeIDs = account => {
   if (
     account &&
     account.admin_office_ids &&
-    !isRoleAvailable(account, 'Super-admin') &&
-    !isRoleAvailable(account, 'State-admin') &&
-    !isRoleAvailable(account, 'County-admin') &&
+    !isRoleAvailable(account, 'Super-admin', 'State-admin', 'County-admin') &&
```
Comment (kelfink) [inline app/javascript/_utils/checkAdminRoles.js]: not equivalent.  The final two parameters are going to be ignored (thanks, javascript for not checking params)

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 2]
```diff
@@ -49,42 +48,31 @@ export default class UserDetail extends Component {
...
+        <div>{userDetailError ? <UserMessage errorMsg={userDetailError} /> : <UserMessage successMsg={saveSuccessMsg} />}</div>
+      )
     return null
   }
 
-  showAddAlert = () => {
-    if (this.props.id) {
-      return (
+  showAddAlert = () => (
+    <div>
```
Comment (kelfink) [inline app/javascript/views/userDetail/UserDetail.jsx]: why not return this?  Don't we need to?

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 3]
```diff
@@ -49,42 +48,31 @@ export default class UserDetail extends Component {
...
+  )
 
-  emailSent = () => {
-    if (this.state.resendEmailAlert) {
-      return <UserMessage successMsg={'Registration email has been sent successfully'} />
-    }
-    return null
-  }
+  emailSent = () => (
+    <div>{this.state.resendEmailAlert ? <UserMessage successMsg={'Registration email has been sent successfully'} /> : null}</div>
```
Comment (kelfink) [inline app/javascript/views/userDetail/UserDetail.jsx]: no return ?

Comment (NagaPradeepGajula-osi) [reply app/javascript/views/userDetail/UserDetail.jsx]: in this case ternary defaultly returns based on the condition. so wouldn't need the return. verified with tests and on UI. the usermessage is being returned.

File: app/javascript/views/userDetail/UserDetail.jsx (hunk 2 above)
Comment (NagaPradeepGajula-osi) [reply app/javascript/views/userDetail/UserDetail.jsx]: that is what we defaultly return 
screen shot 2019-02-13 at 11 24 02 am

File: app/javascript/_utils/checkAdminRoles.js (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply app/javascript/_utils/checkAdminRoles.js]: nice catch. verified with tests and UI was working fine. Didn't thought of the basics will refactor.

Comment (kelfink) [reply app/javascript/_utils/checkAdminRoles.js]: If yarn:test was still working we should probably enhance that.

File: app/javascript/views/userDetail/UserDetail.jsx (hunk 3 above)
Comment (kelfink) [reply app/javascript/views/userDetail/UserDetail.jsx]: Ok.  JS doesn't by default return the last expression evaluated (in my knowledge). so some magic is happening there.

**Pull request merged**